pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -242,6 +242,9 @@ func flowStringToAntreaConnection(flow string, zoneFilter uint16) (*connection.C
 			if err != nil {
 				return nil, fmt.Errorf("conversion of mark '%s' to int failed: %v", fields[len(fields)-1], err)
 			}
+			if uint32(val)&connAllowedCTMarkMask == 0 {
+				break
+			}
```

Comment:
Could there possibly be other marks on the flow that we want to keep?

Reply:
No, but I don't know why I included this in this patch. This check on the mark is already part of `filterAntreaConns`. I will remove this change.